### ISSUE TITLE
Include links to js.xrpl.org from Get Started Using JavaScript Library

### ISSUE THREAD

**Issue by generalburatai** [XRPLF/xrpl-dev-portal#1795]
Are we able to include links to js.xrpl.org in the Docs? That way, devs see the links directly and do not need to go and search for them.

**Comment by intelliot** [comment]
Yes. You can add the links by opening a PR here: https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/tutorials/get-started/get-started-using-javascript.md

**Comment by generalburatai** [comment]
PR here - https://github.com/XRPLF/xrpl-dev-portal/pull/1796

**Comment by amarantha-k** [comment]
There is a link to js.xrpl.org under References: https://xrpl.org/docs/tutorials/javascript/get-started/#see-also

**Comment by mDuo13** [comment]
The Get Started tutorial is now https://xrpl.org/docs/tutorials/javascript/build-apps/get-started and it could still use some cases where it links to the official xrpl.js docs for things like the `Client` class, `Wallet` class, etc.
